examples/js/loaders/MMDLoader.js

```diff
@@ -427,7 +427,7 @@ THREE.MMDLoader.prototype.pourVmdIntoCamera = function ( camera, vmd, name ) {
 
 		var clip = new THREE.AnimationClip( name === undefined ? THREE.Math.generateUUID() : name, -1, tracks );
 
-		if ( clip !== null ) {
```

Comment:
@takahirox FYI lgtm.com reports a few more warnings in `MMDLoader`:
https://lgtm.com/projects/g/mrdoob/three.js/snapshot/bf66a7672bac97558f48a8bec4cfc4bcf0382da4/files/examples/js/loaders/MMDLoader.js